.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
```

Comment:
The `mode` option is set to 'auto'; consider specifying 'debug' or another option explicitly if you know how you will use it, as this can improve clarity on what mode the program is intended to run in.